jetson/ra_kinematics/mrover_arm.cpp

```diff
@@ -562,20 +554,6 @@ void MRoverArm::arm_adjust_callback(std::string channel, ArmAdjustments msg) {
     target_orientation_callback("", target);
 }
 
-void MRoverArm::arm_preset_callback(std::string channel, ArmPreset msg) {
-    std::vector<double> angles = arm_state.get_preset_position(msg.preset);
-    
-    RAPosition new_msg;
-    new_msg.joint_a = angles[0];
-    new_msg.joint_b = angles[1];
-    new_msg.joint_c = angles[2];
-    new_msg.joint_d = angles[3];
-    new_msg.joint_e = angles[4];
-    new_msg.joint_f = angles[5];
-
-    go_to_target_angles(new_msg);
-}
-
 void MRoverArm::encoder_angles_sender() {
```

Comment:
Make this a virtual function and use SAPosition and publish to /sa_position and use joints a b c e.